File: demo2/src/main/java/org/mosquito/demo2/query/OrderItemQuery.java
package org.mosquito.demo2.query;

import org.mosquito.framework.core.model.BaseQuery;

import lombok.Getter;
import lombok.Setter;


/**
 * 分页查询参数实体
 *
 * @author jcchen
 * @Date 2018-5-27
 * 
 */
@Getter
@Setter
public class OrderItemQuery extends BaseQuery {

    private static final long serialVersionUID = 1L;

    /**
     * 
     * 
     */
    private Integer orderId;

    /**
     * 
     * 
     */
    private Integer productId;

    /**
     * 
     * 
     */
    private Integer amount;
	
}

File: demo3/src/main/scala/org/mosquito/demo3/query/OrderItemQuery.scala
package org.mosquito.demo3.query

import org.mosquito.framework.core.model.BaseQuery

import scala.beans.BeanProperty

class OrderItemQuery extends BaseQuery {

  @BeanProperty var orderId: Integer = _

  @BeanProperty var productId: Integer = _

  @BeanProperty var amount: Integer = _
}
